docker-compose.yaml

```diff
@@ -210,10 +228,16 @@ services:
       - POSTGRES_DB=hive
       - POSTGRES_URL=postgres:5432
       - JUPYTERHUB_ADMIN_PASSWORD=testpassword123
+      - NETWORK_NAME=cdm-jupyterhub-network
+      - JUPYTERHUB_USER_IMAGE=cdm-jupyterhub-cdm_jupyterhub:latest
+      - JUPYTERHUB_MOUNT_BASE_DIR=/cdr/cdm/jupyter
```

Comment:
This doesn't look like it's used in this PR

Reply:
yea not in this PR. But used here. https://github.com/kbase/cdm-jupyterhub/blob/main/src/jupyterhub_config/custom_docker_spawner.py#L150

Reply:
Why wasn't this line needed then?

Reply:
to keep PR small, otherwise PR review takes very long.